extra/s6/version

```diff
@@ -0,0 +1 @@
+2.10.0.3 1
```

Comment:
We might want to bump it to 2 so people using s6 from the community repo gets updated.

Reply:
Sure thing.